rust/processor/src/db/common/models/emojicoin_models/constants.rs

```diff
@@ -9,11 +9,19 @@ const STATE: &str = "::emojicoin_dot_fun::State";
 const GLOBAL_STATE: &str = "::emojicoin_dot_fun::GlobalState";
 const LIQUIDITY: &str = "::emojicoin_dot_fun::Liquidity";
 const MARKET: &str = "::emojicoin_dot_fun::Market";
+const ARENA_MELEE: &str = "::emojicoin_arena::Melee";
+const ARENA_ENTER: &str = "::emojicoin_arena::Enter";
+const ARENA_EXIT: &str = "::emojicoin_arena::Exit";
+const ARENA_SWAP: &str = "::emojicoin_arena::Swap";
+const ARENA_VAULT_BALANCE_UPDATE: &str = "::emojicoin_arena::VaultBalanceUpdate";
 
 lazy_static! {
     pub static ref MODULE_ADDRESS: String = std::env::var("EMOJICOIN_MODULE_ADDRESS")
         .expect("EMOJICOIN_MODULE_ADDRESS must be set.")
         .to_owned();
+    pub static ref ARENA_MODULE_ADDRESS: String = std::env::var("EMOJICOIN_ARENA_MODULE_ADDRESS")
```

Comment:
- [ ] Ensure processor does not crash if package is not deployed at the address (e.g. for the hot upgrade then publish strategy)
- [ ] Resolve this comment once verified